examples/llm/inference/convert_hf_to_onnx.py

```diff
@@ -0,0 +1,167 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Basic HuggingFace -> ONNX export script.
```

Comment:
Could you update this to say it also supports exporting mosaicGPT checkpoints stored in HF compatible format?

Reply:
Currently it sounds like it is for exporting HF models to onnx.

Reply:
it should work generally for HuggingFace models, but yes, I will add some language about MosaicGPT